app/src/sharedTest/java/org/oppia/app/player/state/StateFragmentTest.kt

```diff
@@ -617,6 +624,13 @@ class StateFragmentTest {
       )
     ).perform(click())
     onView(withId(R.id.submit_answer_button)).perform(click())
+    onView(
```

Comment:
Please add dedicated tests to make sure that the displaying of the new answer is correct (e.g. make sure that a wrong answer properly uses the recycler view to list all of the selected options.

Reply:
okay got it

Reply:
done

Reply:
Resolving this on behalf of @BenHenning